lib/plottr_components/src/components/timeline/CardCell.js

```diff
@@ -17,7 +18,89 @@ const CardCellConnector = (connector) => {
   const CardAdd = UnconnectedCardAdd(connector)
   const ErrorBoundary = UnconnectedErrorBoundary(connector)
 
-  class CardCell extends PureComponent {
+  class CardCell extends Component {
+    constructor(props) {
+      super(props)
+      this.state = {
+        dragging: false,
+        inDropZone: false,
+        dropDepth: 0,
+        hovering: false, // forces a rerender to determine where to place the overlay
+      }
+    }
+
+    handleDragStart = (e) => {
+      this.setState({ dragging: true })
+      e.dataTransfer.effectAllowed = 'move'
+      const cardIds = this.props.cards.map((card) => card.id)
+
+      if (!Array.isArray(cardIds) && cardIds.length <= 1) return
```

Comment:
Shouldn't this be or (`||`)?
If it's not an array, it surely won't have a length.

Or did you perhaps mean to check whether it *is* an array and has a length less than or equal to 1?

Reply:
Yeah i meant, the second one 😅 but yeah `||` is the correct